pkg/pillar/cmd/zedagent/reportinfo.go

```diff
@@ -650,7 +650,7 @@ func PublishDeviceInfoToZedCloud(ctx *zedagentContext, dest destinationBitset) {
 	// device returns a runtime error. Similarly, we only support enforced application network
 	// interface order for the KVM hypervisor. If enabled for application deployed under Xen
 	// or Kubevirt hypervisor, EVE returns error and the application will not be started.
-	ReportDeviceInfo.ApiCapability = info.APICapability_API_CAPABILITY_SINGLE_STACK_IP_NETWORK
+	ReportDeviceInfo.ApiCapability = info.APICapability_API_CAPABILITY_CELLULAR_ATTACH_CONFIG
```

Comment:
We need handle this properly on controller side if there are devices with different versions

Reply:
What do you mean? We are just expanding APIs with some new configuration fields. Older EVEs will simply not apply them. We do such enhancements all the time.

Reply:
Maybe I'm missing something, but aren't you replacing one reporting capability with another?

Reply:
The capabilities are being added, not replaced, see: https://github.com/lf-edge/eve-api/blob/main/proto/info/info.proto#L695-L696

Reply:
Understood, thank you for the explanation!